Riot/Modules/Room/Members/Detail/RoomMemberDetailsViewController.m

```diff
@@ -884,21 +874,21 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                 case UserEncryptionTrustLevelTrusted:
                 {
                     NSString *info = (self.mxRoom.isDirect) ?
-                    NSLocalizedStringFromTable(@"room_participants_security_information_room_encrypted_for_dm", @"Vector", nil) :
-                    NSLocalizedStringFromTable(@"room_participants_security_information_room_encrypted", @"Vector", nil);
+                    [VectorL10n roomParticipantsSecurityInformationRoomEncryptedForDm] :
+                    [VectorL10n roomParticipantsSecurityInformationRoomEncrypted];
                     [encryptionInformation appendString:info];
                 }
                     break;
                 case UserEncryptionTrustLevelNone:
                     {
                         NSString *info = (self.mxRoom.isDirect) ?
-                        NSLocalizedStringFromTable(@"room_participants_security_information_room_not_encrypted_for_dm", @"Vector", nil) :
-                        NSLocalizedStringFromTable(@"room_participants_security_information_room_not_encrypted", @"Vector", nil);
+                        [VectorL10n roomParticipantsSecurityInformationRoomNotEncryptedForDm] :
+                        [VectorL10n roomParticipantsSecurityInformationRoomNotEncrypted];
                         [encryptionInformation appendString:info];
                     }
                     break;
                 case UserEncryptionTrustLevelUnknown:
-                    [encryptionInformation appendString:NSLocalizedStringFromTable(@"room_participants_security_information_loading", @"Vector", nil)];
+                    [encryptionInformation appendString:[VectorL10n roomParticipantsSecurityLoading]];
```

Comment:
`roomParticipantsSecurityInformationLoading`?

Reply:
Unfortunately we don't have an `InformationLoading` string. We do have `SecurityLoading` which maps to `"Loading…"` which I think is good enough.